components/python/scandir/Makefile

```diff
@@ -0,0 +1,38 @@
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		scandir
+COMPONENT_VERSION=	1.3
+COMPONENT_SUMMARY=	Better directory iterator and faster os.walk(), now in the Python 3.5 stdlib
+COMPONENT_PROJECT_URL=	https://github.com/benhoyt/scandir
+COMPONENT_FMRI=		library/python-2/scandir-27
+COMPONENT_CLASSIFICATION=Development/Python
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=	https://github.com/benhoyt/scandir/archive/v$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH= sha256:006ff89c427b4b8269515a476d2b08881bb2c587d18aebc6a85d6d574d01ae84	
+COMPONENT_LICENSE=	BSD
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/setup.py.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+PYTHON_VERSIONS= 2.7
```

Comment:
It is compatible with 3.4, but was made part of python 3.5, so I would ship it just for 2.7. Once python 3.5 is integrated, there really is no reason to use 3.4.